### PR TITLE
Update cooked

### PR DESCRIPTION
**Pull request by lofcantcook**

### CODE REVIEW

File: cooked [hunk 1]
```diff
@@ -1 +1,2 @@
 cooked
+try hard
```
Comment (KyDurtNewToC) [inline cooked:2]: cannot la

Comment (lofcantcook) [reply cooked:2]: D :

**Review by KyDurtNewToC — APPROVED** [review]
can liao la sha bi

**Review by lofcantcook — COMMENTED** [review]
im cooked

**Pull request merged**